net/interfaces/interfaces.go

```diff
@@ -407,10 +437,10 @@ func GetState() (*State, error) {
 			return
 		}
 		for _, pfx := range pfxs {
-			if pfx.IP().IsLoopback() || pfx.IP().IsLinkLocalUnicast() {
```

Comment:
This means the value of State.HaveV4 still changes on non-Lambda environments.

Maybe we need an isUsableV4 (or just fold both into isUsableIP) and do a host.GetEnvType in that func

Reply:
Added isUsableV4